src/lib/tsreader/TSReader.h

```diff
@@ -80,14 +80,13 @@ namespace MPTV
         long Pause(bool bPaused);
 
         TsReaderState State() { return m_State; };
-
-    private:
-
         /**
          * \brief Translate the given path using the m_basePath setting
          * \param The original (local) timeshift buffer file path on the TV server side

```

Comment:
This is a bit concerning. If this is for the timeshift buffer path why use it for EDL file path?

Reply:
It's a function which converts pvr:....... into the smb:..... path of the actual recording. Mediaportal puts edl files in the same place with the .ts replaced with .edl, so calling this method is the obvious way to find the correct path to open. If there's a different way to establish the correct smb file to open, I'm happy to do that instead.